### DISCUSSION TITLE
Single button on viewing commented Issues

### DISCUSSION THREAD

**Discussion by hokwanhung** [community/community#79280] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

I think that it would be great to include a button on `Issues` to check what I have commented on. Just like the one in Discussions:

<img width="316" alt="image" src="https://github.com/community/community/assets/52453259/e9d61304-2891-40a7-938c-813b34a9847b">

And yes, I know I could use `is:issue commenter:[your_name]`, but why not create a shortcut for it on the Issues page as well - perhaps something like `is:open is:issue commenter:[your_name] archived:false` by default?

<img width="446" alt="image" src="https://github.com/community/community/assets/52453259/de944bf1-9d68-49a7-a3e6-e398423f6957">

**Comment by ghost** [comment]
Yah I too want it

**Comment by Dheeraj-02NK** [comment]
Seems Interesting

**Comment by chmajidnaeem** [comment]
Response #1

**Comment by AMDphreak** [comment]
Strong +1 ? this is still a real friction point.

The search qualifier `commenter:` already works, but the Issues/PRs filter UI exposes **Mentions** while omitting **Commenter** (and **Involves**). That gap makes a common workflow?finding threads you participated in to share as hiring/portfolio links?harder than it needs to be.

I opened a focused write-up with the UI vs search mismatch and those shareable-link use cases here: https://github.com/orgs/community/discussions/202870

Would love to see Commenter (and Involves) land in the filter dropdowns.
